File: src/main/java/swordofmagic7/Command/Developer/GetExp.java
package swordofmagic7.Command.Developer;

import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;
import swordofmagic7.Command.SomCommand;
import swordofmagic7.Data.PlayerData;

public class GetExp implements SomCommand {
    @Override
    public boolean PlayerCommand(Player player, PlayerData playerData, String[] args) {
        if (args.length >= 1) {
            try {
                playerData.addPlayerExp(Integer.parseInt(args[0]));
            } catch (Exception e) {
                player.sendMessage("§c" + "/getExp <exp>");
            }
        }
        return true;
    }

    @Override
    public boolean Command(CommandSender sender, String[] args) {
        return false;
    }
}

File: src/main/kotlin/swordofmagic7/Command/Developer/GetExp.kt
package swordofmagic7.Command.Developer

import org.bukkit.command.CommandSender
import org.bukkit.entity.Player
import swordofmagic7.Command.SomCommand
import swordofmagic7.Data.PlayerData

class GetExp : SomCommand {
    override fun PlayerCommand(player: Player, playerData: PlayerData, args: Array<String>): Boolean {
        if (args.isNotEmpty()) {
            try {
                playerData.addPlayerExp(args[0].toIntOrNull() ?: throw IllegalArgumentException("Invalid exp value"))
            } catch (e: Exception) {
                player.sendMessage("§c/getExp <exp>")
            }
        }
        return true
    }

    override fun Command(sender: CommandSender, args: Array<String>): Boolean {
        return false
    }
}
